bind.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "bind",
+    "release": "11.3-RELEASE",
+    "artifact": "https://github.com/pandorixtech/iocage-plugin-bind.git",
+    "official": false,
+    "properties": {
+        "nat": 1,
+        "nat_forwards": "udp(53:53)"
+    },
+    "pkgs": [
+        "bind916",
+	"vim",
+	"mc"
```

Comment:
Is `vim` and `mc` really required by the bind package? I couldn't see any usages of it in your `post_install.sh` either.

Also the JSON indent got a bit weird in this pkgs array :)

Reply:
Thank you for your review, the format has been corrected. 
Although vim and mc are not required to run bind, since the configuration is done by editing configuration files, I decided to add the most common tools used by administrators for that, vim editor and midnight commander. If that is an issue to accept this plugin I will remove them.

Reply:
No problem, you can keep them here =) Was more just wondering if it was a requirement for something else.